appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/util/ConnectionPoolReconfigHelper.java

```diff
@@ -147,7 +147,7 @@ private static ReconfigAction isEqualConnectorConnectionPool(ConnectorConnection
                 && !(newCcp.getDataStructureParameters().equals(oldCcp.getDataStructureParameters()))) {
             return ReconfigAction.RECREATE_POOL;
         }
-
+        
```

Comment:
I have the same issue that I've noticed.  Not sure if it's adding or deleting space.  Something has to do with NB.